draft-ietf-mls-protocol.md

```diff
@@ -911,7 +907,9 @@ an attacker that had compromised the sender's prior leaf secret.
 Update messages SHOULD be sent at regular intervals of time as long as the group
 is active, and members that don't update SHOULD eventually be removed from the
 group. It's left to the application to determine an appropriate amount of time
-between Updates.
+between Updates.  In general, however, applications should take care that they
+do not send MLS messages at a rate that overwhelms the transport over which
+messages are being sent.
```

Comment:
/transport/ or /network path/

Reply:
Here we are using "transport" in a higher-level sense.  As @Bren2010 points out, the "transport" could be something like WebSockets or Tor cells.  It is possible in principle that you might use something like UDP or raw IP packets, but that would require a lot of invention.

Reply:
I think the name "update message" is perhaps causing some confusion. They are not some kind of keepalive packets or high-frequency state synchronization messages. In many applications, the interval between two update messages can range from hours to months.
Their primary purpose is to limit the "window of compromise" by providing post-compromise security, I doubt applications will require the window to be < 1 min.